consolidator/Dockerfile

```diff
@@ -12,10 +16,17 @@ ARG consolidator_config_url
 ARG max_heap_size=2048m
 ARG ballerina_download_url=https://dist.ballerina.io/downloads/2201.0.0/ballerina-2201.0.0-swan-lake-linux-x64.deb
 ENV consolidator_config_file_url_env=${consolidator_config_url}
+
+WORKDIR /home/${container_user}
+
 COPY ./target/bin/*.jar consolidator.jar
 EXPOSE 9192
-RUN wget -q --show-progress ${ballerina_download_url} -O ballerina-linux-installer-x64.deb
-RUN dpkg -i ballerina-linux-installer-x64.deb
+RUN groupadd -r -g ${container_user_gid} ${container_user_group} \
+&&  useradd -r -u ${container_user_uid} -g ${container_user_group} ${container_user} \
+&& wget -q --show-progress ${ballerina_download_url} -O ballerina-linux-installer-x64.deb \
+&& dpkg -i ballerina-linux-installer-x64.deb \
+&& chown -R ${container_user}:${container_user} /home/${container_user}
 #TODO Link to be parameterized instead of hardcoding
+USER ${container_user_uid}:${container_user_gid}
 CMD wget -q --show-progress "${consolidator_config_file_url_env}" -O Config.toml;\
-     java -jar -Xms256m -Xmx2048m ./consolidator.jar ;\
+     java -jar -Xms256m -Xmx${max_heap_size} ./consolidator.jar ;
```

Comment:
set both heap memory `-Xms256m -Xmx${max_heap_size}` to be an env variable

Also in line no. 16  only ARGS has been provided `max_heap_size`. set it as an env variable

Reply:
updated